notify/api/v1/api.py

```diff
@@ -91,8 +91,8 @@ def send_notification(backends):
                 result["result"][backend][drv_name] = {"error": str(e)}
                 result["errors"] += 1
             except Exception as e:
-                LOG.error(("Backend '{}' driver '{}' "
-                           "error: {}").format(backend, drv_name, e))
+                LOG.error("Backend '{}' driver '{}': {}: {}".format(
```

Comment:
`logging` itself formats the string and you should not do it explicitly because `logging` does it according to the specified level and can skip the formatting if it is not necessary. So, instead of 

```
    LOG.error("Backend '{}' driver '{}': {}: {}".format(backend, drv_name, type(e), e))
```

you should do:

```
        LOG.error("Backend '%s' driver '%s': %s: %s", backend, drv_name, type(e), e)
```

Unfortunately, `logging` does not support the format style by default.